scm/github/repo.go

```diff
@@ -346,6 +346,12 @@ func (c *client) ListUserRepos(u *library.User) ([]*library.Repo, error) {
 
 	// iterate through each repo for the user
 	for _, repo := range r {
+		// skip if the repo is void
+		//   fixes an issue with GitHub replication being out of sync
```

Comment:
Remove extra whitespace
```suggestion
		// fixes an issue with GitHub replication being out of sync
```

Reply:
resolved, thank you